app.js

```diff
@@ -0,0 +1,61 @@
+/* jshint esversion:6*/
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+const json = bodyParser.json();
+const urlParser = bodyParser.urlencoded({ extended: false });
+const fail = {success: false};
+const success = {success:true};
+const heard = {heard: false};
+const bigBuzz = [];
+let userScore = {score : 0};
+
+app.use(bodyParser.urlencoded());
```

Comment:
So you already have a bodyParser defined on line 7 but you're making a new one here.  This seems a bit redundant. Also the version on line 7 is actually more complete in this scenario.